refinery/src/runner.rs

```diff
@@ -60,7 +60,7 @@ impl Migration {
     }
 
     pub fn checksum(&self) -> u64 {
-        let mut hasher = DefaultHasher::new();
+        let mut hasher = SipHasher13::new();
```

Comment:
can we just add a comment above here stating that we changed to siphash due to DefaultHasher's unreliability across versions because of the hasher algorithm omission, and that we choose siphash to maintain compability with the current one?

Reply:
I've added a comment here in the latest commit.